src/main.cpp

```diff
@@ -190,6 +191,11 @@ void init() {
     #ifndef NO_UTILS_MOTORDRIVERCONTROL
     kernel->add_module( new MotorDriverControl(0) );
     #endif
+
+    auto rpp = new ReadPinPool();
```

Comment:
This needs to be protected with `#ifndef NO_UTILS_READPIN` like all the other modules.